.ng-dev/commit-message.mts

```diff
@@ -31,6 +31,7 @@ export const commitMessage: CommitMessageConfig = {
     'cdk/overlay',
     'cdk/platform',
     'cdk/portal',
+    'cdk/private',
```

Comment:
It's up for debate whether we should have another entrypoint for this, but I couldn't where it would fit better. We can also use this now to move some of the private utilities from `cdk/platform`.

Reply:
Hmm...seems equivalent to what we had to do with `material/core/private`. I think it makes sense to have a private package and make it clear to users that its not part of our API (and eligible for breaking changes at any tyime)